src/main/java/org/apache/maven/doxia/DefaultConverter.java

```diff
@@ -248,14 +249,21 @@ public void convert(InputFileWrapper input, OutputFileWrapper output)
                 for (File f : files) {
                     File relativeOutputDirectory = new File(
                             PathTool.getRelativeFilePath(input.getFile().getAbsolutePath(), f.getParent()));
-                    parse(f, input.getEncoding(), input.getFormat(), output, relativeOutputDirectory);
+                    convert(f, input.getEncoding(), input.getFormat(), output, relativeOutputDirectory);
+                    cleanUp(input, f);
                 }
             }
         } finally {
             stopPlexusContainer();
         }
     }
 
+    private void cleanUp(InputFileWrapper input, File f) {
+        if (input.cleanUp(f)) {
+            LOGGER.info("Removed input file \"{}\" after successfull conversion", f);
```

Comment:
Isn't that too verbose at info level?

Reply:
For me this is worth to be always emitted as when giving a directory as input parameter it is otherwise not clear which files within are removed.

Reply:
Accepted.